File: FromSlides/Fortran/01_hello_world.f90
PROGRAM hello_world

  USE omp_lib

  PRINT *, 'Hello world from the main program'

  !$OMP PARALLEL

    PRINT '(A,I0)','Hello world! I am processor ', omp_get_thread_num()

  !$OMP END PARALLEL

END PROGRAM hello_world

File: FromSlides/C/01_hello_world.c
#include <stdio.h>
#include <omp.h>

int main()
{
  printf("Hello world from the main program\n");
#pragma omp parallel
  {
    printf("Hello world! I am processor %i\n", omp_get_thread_num());
  }
  return 0;
}

